api/service.py

```
@@ -11,7 +11,10 @@
 from collections import deque
 from fastapi import FastAPI, HTTPException, status
 from fastapi.middleware.cors import CORSMiddleware
+from fastapi.security import HTTPBasic, HTTPBasicCredentials
+from fastapi import FastAPI, HTTPException, status, Depends
```

Ruff (0.14.10)

Redefinition of unused `FastAPI` from line 12

Remove definition: `FastAPI`

(F811)